apps/api/src/server/repositories/case.repository.js

```diff
@@ -301,6 +304,14 @@ const updateApplicationSansRegionsRemoval = ({
 						}
 					}
 				}),
+			...(caseStatus && {
+				CaseStatus: {
+					updateMany: {
```

Comment:
I thought that CaseStatus kept an audit history of all the status changes on a case, with the valid = 1 indicating the latest record. .  This code updates all status records for that case to the new value.